Update assign-policies-users-and-groups.md (#8755)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/assign-policies-users-and-groups.md

#### Group assignment ranking

> [!NOTE]
> A given policy type can be assigned to at most 64 groups, across policy instances for that type.

Reviewer: ```suggestion
> [!NOTE]
A given policy type can be assigned to a maximum of 64 groups across policy instances for that type.
```
